docs/src/main/sphinx/release/release-361.md

```diff
@@ -0,0 +1,88 @@
+# Release 361 (XX Aug 2021)
+
+## General
+
+* Support table redirection for `INSERT`, `DELETE` and `UPDATE` operations. ({issue}`8683`)
+* Improve performance of {func}`sum` and {func}`avg` aggregations involving `decimal` values. ({issue}`8878`)
+* Improve performance for queries using `IN` predicate with moderate to large number of constants. ({issue}`8833`)
+* Fix failures of specific queries accessing ROW columns with fields requiring a name quoting.  ({issue}`8845`)
+
+## Security
+
+* Add support for OAuth2/OIDC opaque access tokens. The property `http-server.authentication.oauth2.audience` has 
+  been removed in favor of using `http-server.authentication.oauth2.client-id` as expected by OIDC. To support 
+  audiences which are not the `client-id` the new property `http-server.authentication.oauth2.additional-audiences` 
+  has been added. Additionally the new property `http-server.authentication.oauth2.issuer` is required; tokens 
+  which are not issued by this URL will be rejected. ({issue}`8641`)
+
+## Cassandra connector
+
+* Add support for pushdown of `smallint`, `tinyint` and `date` types on partition columns. ({issue}`3763`)
+* Fix incorrect results for predicates with inequality on clustering key. ({issue}`401`)
+
+## ClickHouse connector
+
+* Add new configuration property `metadata.cache-maximum-size` to allow limiting the size of the metadata cache. ({issue}`8652`)
+* Fix incorrect results for aggregation functions applied to columns of type VARCHAR and CHAR. ({issue}`7320`)
+
+## Druid connector
+
+* Add new configuration property `metadata.cache-maximum-size` to allow limiting the size of the metadata cache. ({issue}`8652`)
+
+## Hive connector
+
+* Allow access to `$properties` table for Trino and Hive views. ({issue}`8805`)
+* Add support for translating Hive views which use `from_utc_timestamp` function. ({issue}`8502`)
+* Improve performance of updating Glue table statistics for partitioned tables. Also change default 
+  Glue statistics read/write parallelism from 1 to 5. ({issue}`8839`)
+* Improve performance of querying Parquet data. ({issue}`7349`)
```

Comment:
Generally? Or can we be more specific?

Reply:
It's hard to describe without getting into the weeds. It's for parquet files that contain column indexes, when the query has a filter that can be satisfied by the column indexes. Most users would never know that a file has column indexes, though.